packages/cli/lib/resources/template.html

```diff
@@ -10,9 +10,10 @@
 		<% if (htmlWebpackPlugin.options.manifest.theme_color) { %>
 			<meta name="theme-color" content="<%= htmlWebpackPlugin.options.manifest.theme_color %>">
 		<% } %>
-		<% for (const file in htmlWebpackPlugin.options.createLoadManifest(compilation.assets)[htmlWebpackPlugin.options.url]) { %>
-			<% if (file) { %>
-				<link rel="preload" href="<%= htmlWebpackPlugin.files.publicPath + file %>" as="<%= file.match(/\.css$/)?'style':'script' %>" <%= file.match(/\.esm\.js$/)?'crossorigin="anonymous"':'' %>>
+		<% for (const file in htmlWebpackPlugin.options.createLoadManifest(compilation.assets, htmlWebpackPlugin.options.config.esm)[htmlWebpackPlugin.options.url]) { %>
+			<% if (file && file.match(/\.(css|js)$/)) { %>
+				<!-- crossorigin for main bundle as that is loaded from <script type=module tag, other lazy loaded bundles are from webpack so its not needed -->
```

Comment:
probably can skip the comment or put it in `<% /* these gross tags */ %>`

Reply:
done